app/src/main/res/values/strings.xml

```diff
@@ -184,6 +184,7 @@
     <string name="action_deep_link" translatable="false">
         org.gdg.frisbee.android.DEEP_LINK_ACTION
     </string>
+  <string name="invitation_error_message">App Invite was unsuccessful.</string>
```

Comment:
I wanted to enable the error in AppInvite flow. But this introduced another String. Do you think we can use one of the existing ones.